src/ffi.rs

```diff
@@ -3,27 +3,36 @@
 #![allow(unused)]
 
 use std::mem;
-use std::os::raw::{c_char, c_double, c_int, c_longlong, c_uchar, c_uint, c_ulonglong, c_ushort, c_void};
+use std::os::raw::{
+    c_char, c_double, c_int, c_longlong, c_uchar, c_uint, c_ulonglong, c_ushort, c_void,
+};
 use std::ptr;
 
 pub type lua_Integer = c_longlong;
 pub type lua_Unsigned = c_ulonglong;
 pub type lua_Number = c_double;
 
+#[derive(Debug)]
```

Comment:
This seems wrong to me - given that `lua_State` is, as far as the Rust type system is concerned, an empty enum so can't have any instances.

Reply:
I will remove it.